ext/hiera/hiera.yaml

```diff
@@ -0,0 +1,15 @@
+---
+version: 5
+defaults:
+  # The default value for "datadir" is "data" under the same directory as the hiera.yaml
+  # file (this file)
+  # When specifying a datadir, make sure the directory exists.
+  # See https://docs.puppet.com/puppet/4.10/environments.html for further details on environments.
+  # datadir: data
+  data_hash: yaml_data
```

Comment:
This is already the default so it's inconsistent to comment out `datadir: data` and leave this line uncommented.